src/compiler/xquery/compile/compile-scenario.xsl

```diff
@@ -17,12 +17,11 @@
       <xsl:param name="call" as="element(x:call)?" required="yes" tunnel="yes" />
       <xsl:param name="context" as="element(x:context)?" required="yes" tunnel="yes" />
       <xsl:param name="pending" as="node()?" required="yes" tunnel="yes" />
+      <xsl:param name="pending-p" as="xs:boolean" />
+      <xsl:param name="run-sut-now" as="xs:boolean" />
```

Comment:
Do you want to add `required="yes"`, because an empty sequence isn't valid for `xs:boolean`? I have the same question in these other files in this PR:
```
xspec/src/compiler/xquery/compile/compile-expect.xsl
xspec/src/compiler/xslt/compile/compile-scenario.xsl
xspec/src/compiler/xslt/compile/compile-expect.xsl
```

Reply:
Thanks for looking into this.
`require="yes"` will help static analysis. I added it throughout the repository in d695d4fcf63a8f70e161f368c799ce09c9463a57 and 42209d90e5d5b1576455cab902afbd29c49fa4f6.

Reply:
You went above and beyond. :)